use_cases/eluc/.pylintrc

```diff
@@ -1,10 +1,14 @@
 [MASTER]
-ignore=demo
+ignore=demo, prescriptors/esp
+
+recursive=y
+
+fail-under=9.0
```

Comment:
Moved flags from eluc.yml into here. Now we ignore the esp directory because it can't be installed in the Github environment